### PR TITLE
Actualize platform Nuget packages. Up Newtonsoft.Json to 12.0.2.0

### PR DESCRIPTION
**Pull request by trueboroda**
### Problem
https://github.com/VirtoCommerce/vc-platform/issues/1719
For the new actual version of Elasticsearch, we need to use Newtonsoft.Json version 12.0.0 and above. 

### Solution
Platform Nuget packages were updated to the latest in the scope of the current major version of every package.
Newtonsoft.json was updated to 12.0.2

### CODE REVIEW

File: VirtoCommerce.Platform.Web/Cache/RedisCacheBackplane2.cs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿using System;
+using System;
```
Comment (yecli) [inline VirtoCommerce.Platform.Web/Cache/RedisCacheBackplane2.cs]: Please revert this file completely

File: VirtoCommerce.Platform.Web/Cache/RedisConnectionManager2.cs [hunk 2]
```diff
@@ -1,4 +1,4 @@
-﻿using System;
+using System;
```
Comment (yecli) [inline VirtoCommerce.Platform.Web/Cache/RedisConnectionManager2.cs]: Please revert this file completely

Comment (yecli) [reply VirtoCommerce.Platform.Web/Cache/RedisConnectionManager2.cs]: This will allow eliminating sonar errors.

Comment (trueboroda) [reply VirtoCommerce.Platform.Web/Cache/RedisConnectionManager2.cs]: It is done

File: VirtoCommerce.Platform.Web/Web.config [hunk 3]
```diff
@@ -31,12 +32,12 @@
     </nlog>
     <connectionStrings>
         <!-- Database Connection String -->
-        <add name="VirtoCommerce" connectionString="Data Source=(local);Initial Catalog=VirtoCommerce2;Persist Security Info=True;User ID=virto;Password=virto;MultipleActiveResultSets=True;Connect Timeout=420" providerName="System.Data.SqlClient" />
+        <add name="VirtoCommerce" connectionString="Data Source=(local);Initial Catalog=VirtoCommerce2_2;Persist Security Info=True;User ID=virto;Password=virto;MultipleActiveResultSets=True;Connect Timeout=420" providerName="System.Data.SqlClient" />
```
Comment (yecli) [inline VirtoCommerce.Platform.Web/Web.config]: Please revert this change

File: VirtoCommerce.Platform.Web/Web.config [hunk 4]
```diff
@@ -31,12 +32,12 @@
...
         <!-- Assets Connection String -->
         <!--<add name="AssetsConnectionString" connectionString="provider=AzureBlobStorage;DefaultEndpointsProtocol=https;AccountName=yourAccountName;AccountKey=yourAccountKey" />
             or with CDN (optional)
             <add name="AssetsConnectionString" connectionString="provider=AzureBlobStorage;DefaultEndpointsProtocol=https;AccountName=yourAccountName;AccountKey=yourAccountKey;cdnUrl=yourStorageName.azureedge.net" />-->
-        <add name="AssetsConnectionString" connectionString="provider=LocalStorage;rootPath=~/App_Data/Assets;publicUrl=http://localhost/admin/assets" />
+        <add name="AssetsConnectionString" connectionString="provider=LocalStorage;rootPath=~/App_Data/Assets;publicUrl=http://localhost/admin2/assets" />
```
Comment (yecli) [inline VirtoCommerce.Platform.Web/Web.config]: Please revert this change

**Comment by t13ka** [comment]
it seems that all is ok
